npm/webpack-dev-server/src/devServer.ts

```diff
@@ -24,6 +24,7 @@ export type WebpackDevServerConfig = {
 } & {
   framework?: typeof ALL_FRAMEWORKS[number] // Add frameworks here as we implement
   webpackConfig?: unknown // Derived from the user's webpack
+  options?: ProjectConfig
```

Comment:
I'd like this to be:
```
options?: {
  projectConfig: ProjectConfig
}
```

That way we can encapsulate all of the project config options into one property. If we want to add another option in the future, the interfaces don't have to merge.

Also, we will need to work on types a bit since this type should only apply to projects that have specified `angular`.

Reply:
Great idea! I'll add this in